portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,95 @@
-#content {
+.content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  margin-bottom: 20px;
+  width: 75%;
 }
 
-#greeting-container {
+.projpic {
+  width: 30%;
+  padding-left: 10px;
+  padding-right: 10px;
   margin-top: 20px;
+  margin-bottom: 20px;
+}
+.randomFact {
+  margin-top: 20px;
+  margin-bottom: 30px;
 }
 
-.profilePic {
+.profile{
+  margin-right: 20px;
+  float: left;
+  text-align: center;
+}
+
+#profilePic {
   display: block;
   margin-right: auto;
   margin-left: auto;
-  width: 40%;
+  width: 30%;
+  height: auto;
+}
+
+/* Add a black background color to the top navigation */
+.nav {
+  background-color: #333;
+  overflow: hidden;
 }
+
+/* Style the links inside the navigation bar */
+.nav a {
+  float: left;
+  display: block;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.nav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/* Add an active class to highlight the current page */
+.nav a.active {
+  background-color: #4CAF50;
+  color: white;
+}
+
+/* Hide the link that should open and close the nav on small screens */
+.nav .icon {
+  display: none;
+  font-size: 15px;
+}
+
+/* When the screen is less than 600 pixels wide, hide all links, 
+ * except for the first one ("Home"). 
+ * Show the link that contains should open and close the nav (.icon) */
+@media screen and (max-width: 600px) {
+  .nav a:not(:first-child) {display: none;}
+  .nav a.icon {
+    float: right;
+    display: block;
+  }
+}
+
+/* The "responsive" class is added to the nav with JavaScript 
+ * when the user clicks on the icon. This class makes 
+ * the nav look good on small screens (display the links vertically) */
+@media screen and (max-width: 600px) {
+  .nav.responsive {position: relative;}
+  .nav.responsive a.icon {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  .nav.responsive a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
```

Comment:
Your brackets don't match. Perhaps you should check using Chrome developer tools that your CSS and JavaScript don't generate any errors.

Reply:
Thanks for catching it, missing a closing bracket at the end. Fixing it.

Reply:
The bracket is missing again. I think that you might have missed including one commit.